test/__snapshots__/validate-options.test.js.snap.webpack5

```diff
@@ -949,7 +948,6 @@ exports[`options validate should throw an error on the "webSocketServer" option
    Details:
     * options.webSocketServer should be one of these:
       false | \\"sockjs\\" | \\"ws\\"
-      -> Deprecated: please use '--web-socket-server-type' option.
```

Comment:
Hm, why it was removed?

Reply:
Because only CLI flag is deprecated, so only adding this description to CLI flag description and CLI help output.

Reply:
:+1: Sorry, missing this